iOS/DuckDuckGo/AIChat/AIChatViewControllerManager.swift

```diff
@@ -94,6 +97,7 @@ final class AIChatViewControllerManager {
 
         pixelMetricHandler = AIChatPixelMetricHandler(timeElapsedInMinutes: sessionTimer?.timeElapsedInMinutes())
         pixelMetricHandler?.fireOpenAIChat()
+        featureDiscovery.setWasUsedBefore(.aiChat)
```

Comment:
This is the key change, now we set `wasUsedBefore` any time the Duck.ai VC manager opens the flow.